config/jobs/kubernetes-sigs/secrets-store-csi-driver/secrets-store-csi-driver-config.yaml

```diff
@@ -199,6 +199,41 @@ presubmits:
       testgrid-tab-name: pr-secrets-store-csi-driver-e2e-windows
       description: "Run E2E tests on a Windows cluster for Secrets Store CSI driver."
       testgrid-num-columns-recent: '30'
+  - name: pull-secrets-store-csi-driver-e2e-gcp
+    decorate: true
+    decoration_config:
+      timeout: 25m
+    always_run: true
```

Comment:
Should we start off as `optional: true`, set it as required once https://github.com/kubernetes-sigs/secrets-store-csi-driver/pull/340 is merged and tests are green? Otherwise it can block PRs from merging.

Example for windows tests - https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/secrets-store-csi-driver/secrets-store-csi-driver-config.yaml#L150

Reply:
Ah yes good point. I've added the `optional: true` but left `always_run: true`, is that right or should that be set to false?

Reply:
`optional: true` with `always_run: true` is good. Once we have successful test runs, we can remove optional.